code/modules/mob/living/simple_animal/friendly/axolotl.dm

```diff
@@ -23,6 +23,8 @@
 	can_be_held = TRUE
 	held_w_class = WEIGHT_CLASS_TINY
 	worn_slot_flags = ITEM_SLOT_HEAD
+	held_lh = 'icons/mob/inhands/animal_item_lh.dmi'
+	held_rh = 'icons/mob/inhands/animal_item_rh.dmi'
```

Comment:
Can you expand these to lefthand and righthand? Some files are like that in the folder, some aren't, but we generally do not abbreviate names anymore.